docs/moonbase/devices.md

```
@@ -2,9 +2,8 @@
 
 <img width="320" src="https://github.com/user-attachments/assets/dacd1527-b771-4d57-8528-5d42d89e3cac" />
 
-Sends and receives UDP messages over the network to see which other devices are there (WLED compatible). 
+Shows which other MoonLight devices are connected to your local network.
 
-* Device name: name of this device
+* Device name: name of this device (set the name in the [WiFi station module](https://moonmodules.org/MoonLight/network/sta/))
 * Devices: Devices found on the network
-    * Click on IP to jump to the device
-* Statusbar and HTML title display the device name
+    * Click on IP to jump to the device
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)